sycl/test-e2e/Graph/Inputs/debug_print_graph.cpp

```diff
@@ -45,12 +45,12 @@ int main() {
       CGH.copy(AccB, AccA);
     });
 
-    add_node(Graph, Queue, [&](handler &CGH) {
-      auto AccA = BufferA2D.get_access<access::mode::read>(CGH);
```

Comment:
Ca we remove `BufferA2D` all together from lines 30 & 31?

Reply:
I removed all unnecessary variables: BufferA2D among others.